providers/common.rb

```diff
@@ -0,0 +1,71 @@
+include_recipe 'git'
```

Comment:
Basically, you want `run_context.include_recipe`. The link in my first comment has more details.